### DISCUSSION TITLE
I need to convert a playingMessage to embed message. Please help me

### DISCUSSION THREAD

**Discussion by arshalabbas** [eritislami/evobot#238] [Ideas] [answered]
I provided the code down, let's check it out...and help me. PLEASE

`var playingMessage = await queue.textChannel.send(`🎶 Started playing: **${song.title}**`);`

In Include>play.js
Please convert this message to embed message.

**Comment by brandonst** [comment, marked as answer]
making it into an embed is very simple, to start, add this as the first line of that file if it isn't already there
`const Discord = require('discord.js')`

Then replace `var playingMessage = await queue.textChannel.send(🎶 Started playing: ${song.title})`

with
```js
const playingEmbed = new Discord.MessageEmbed()
    .setDescription(`🎶 Started playing: ${song.title}`)
    
let playingMessage = await queue.textChannel.send(playingEmbed)
```

Now, this is a very simple embed that only includes a description,
and I'm assuming you want to expand it to include more information.
If you wish to add more things like a title, or author, refer to this guide
https://discordjs.guide/popular-topics/embeds.html

**Reply by arshalabbas** [reply]
Thank you very much, it's working...

**Reply by mihawkman** [reply]
```
const playingEmbed = new Discord.MessageEmbed()
    .setColor(`f82a2a`)
	.setTitle(`:headphones: Started playing: **${song.title}**`)
  .setDescription(`<${song.url}>`)
    
let playingMessage = await queue.textChannel.send(playingEmbed)
```
I've changed the code a little bit... But it looks like the media controls via reactions doesn't work. Anyone know why its not working? Tq

**Reply by ghost** [reply]
You forgot a # in the .setColor

**Reply by mihawkman** [reply]
> You forgot a # in the .setColor

it still work without the #....I have a new problem though....message pruning doesn't work anymore if you changed it to an embed message

**Reply by Prasham27** [reply]
@mihawkman can u help me with the bot, I have one more query

**Reply by arshalabbas** [reply]
> @mihawkman can u help me with the bot, I have one more query

What happend?
Maybe I can help you.

**Reply by Prasham27** [reply]
@arshalabbas I want the bot not to leave vc and stay there for 24/7, what changes do I need to make in the code?

**Reply by arshalabbas** [reply]
Discord : keep__lol#5722

@PrashLucifer

**Reply by karimawi** [reply]
@brandonst Do you know how I can fetch the youtube video thumbnail to put it as the embed's thumbnail?
